lib/screens/repositories_screen.dart

```diff
@@ -0,0 +1,65 @@
+import 'package:flutter/material.dart';
+
+import '../widgets/drop_down.dart';
+
+class RepositoriesScreen extends StatefulWidget {
+  const RepositoriesScreen({super.key});
+
+  @override
+  State<RepositoriesScreen> createState() => _RepositoriesScreenState();
+}
+
+class _RepositoriesScreenState extends State<RepositoriesScreen> {
+  final List<String> _languages = [
+    "Dart",
+    "Java",
+    "JavaScript",
+    "Kotlin",
+    "PHP",
+    "Python",
+    "Ruby",
+    "Swift",
+    "TypeScript",
+  ];
+  String? _selectedLanguage;
+  List<String> repositories = [];
+
+  void _didSelectItem(String item) {
+    setState(() {
+      _selectedLanguage = item;
+      repositories = List<String>.generate(100, (index) => "Repo ${index + 1}");
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) => Scaffold(
+        appBar: AppBar(
+          title: const Text("Repositories"),
+        ),
+        body: Container(
```

Comment:
The Container widget is quite heavy and is not so performant as other Widgets. In this case I believe it could be replaced with the Padding widget?

Reply:
I would always try to avoid using that widget unless it is ur only way.

Reply:
done